xtask/src/ci.rs

```diff
@@ -189,4 +187,14 @@ impl CiTask {
         .run()
         .map_err(Into::into)
     }
+
+    /// Check the typos.
+    fn typos(&self) -> Result<()> {
+        if cmd!("which typos").run().is_err() {
```

Comment:
Let's do this the same way as in the SDK and wait for upstream improvements. `which` isn't necessarily available on every system, especially Windows would probably be broken here (and on arch linux it's also not a core package IIRC).

Reply:
I could just check `typos --version` then

Reply:
Hm okay.